### PR TITLE
Feature/welcome page build

### PR DESCRIPTION
**Pull request by AshleeAWebb**
# Checklist:

## Type of change:

   - [x] New feature
   - [] Bug Fix
   - [] Refactor

## Description of Changes:

    Description: I've added the web browser logo to the icon and created a welcome page with some programming logic. The router switch is being built, and the buttons respond to clicks. I think it's a good start, the redirect function works well.

## Requested feedback: 
    
    I want feedback on...the use of hooks to set the user will most likely need to change once we get the mock data but wanted to attempt something similar to get us started I can always delete it.

## Check the correct boxes:

   - [x] This broke nothing
   - [] This broke some stuff
   - [] This broke everything

## Testing Changes:

   - [x] No Tests have been changed
   - [] Some Tests have been changed
   - [] All of the Tests have been changed(Please describe what in the world happened)

## Checklist:
   - [] All Tests are Passing
   - [] My code has no unused/commented out code
   - [x] I have reviewed my code
   - [] I have commented on my code, particularly in hard-to-understand areas
   - [] I have fully tested my code
   - [x] I have added one or more people as "Reviewers" to review & merge

(For Fun!)Please Include a link to a gif of your feelings about this branch

Link:

ghost

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/diceicon.ico" />
```
Comment (asutherland91) [inline public/index.html:5]: This is a great touch!

**Review by asutherland91 — COMMENTED** [review]
I agree that the use of hooks will likely need to change, but for now it looks great! Fantastic job!

File: src/Components/ReusableComponents/Button/Button.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    0% {
+      box-shadow: 0 0 5px #fefefe;
+    }
+    50% {
+      box-shadow: 0 0 30px 5px #fefefe;
+    }
+    100% {
+      box-shadow: 0 0 5px #fefefe;
+    }
+  }
```
Comment (tdmburr) [inline src/Components/ReusableComponents/Button/Button.css:25]: Appreciate the work on the glow effect here.

**Review by tdmburr — APPROVED** [review]
This all looks great! Nice and clean code.

**Pull request merged**